src/Microsoft.Extensions.Caching.StackExchangeRedis/baseline.netcore.json

```diff
@@ -0,0 +1,295 @@
+{
```

Comment:
Brand new packages shouldn't have baselines, should they?

Reply:
"Brand new", it shouldn't have any API differences from the 2.2 package so we should be copying it over so api check verifies it.

Reply:
Ah, nifty.